### PR TITLE
Refactoring initial content

### PR DESCRIPTION
**Pull request by newOnahtaN**
Cleaned up some changes suggested by Ken and Karl. Removed a lot of boilerplate rails code, removed references to any kind of database, made relevant scripts more 'ruby-like'.

### CODE REVIEW

File: app/models/delivered_stories_report.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      end.flatten!.compact
+  end
+
+  def get_time_of_last_deployment
+    app_name = ENV["MAIN_APP_NAME"]
+    heroku  = Heroku::API.new(:api_key => ENV["HEROKU_API_KEY"])
+    @deployment_date = heroku.get_releases(app_name).body.last['created_at']
+  end
+
+  def get_pivotal_api_token
```
Comment (kenliu) [inline app/models/delivered_stories_report.rb]: Just a very minor ruby style comment...usually ruby code doesn't use methods named `get_something` for reading an instance varaible, you would just create a method called `something`. So this case you name the method `pivotal_api_token` and reference it as if it were a local variable.

**Comment by kbaum** [comment]
Does this email get sent out automatically when we deploy?

—

**Comment by newOnahtaN** [comment]
No, not yet. There are two things that still need to be done. 
(1) Environment variables still need to be set in the heroku app named pivotal-deployment-emailer:

```
Currently:
EMAIL_RECIPIENT = my email
HEROKU_API_KEY = my api key
MAIN_APP_NAME = vts-dev

Needs to be:
EMAIL_RECIPIENT = an email that serves as a forwarding address for everyone who wants these emails
HEROKU_API_KEY = an api key that has access to the main app
MAIN_APP_NAME = the name of the main production app in heroku
```

(2) The command `heroku run rake email_delivered_stories --app pivotal-deployment-emailer` needs to be added somewhere to the deployment script.

I can do the (2), but (1) requires information I do not have access to.

**Comment by kenliu** [comment]
I will set this up

On Monday, June 29, 2015, Nathan Owen notifications@github.com wrote:

> No, not yet. There are two things that still need to be done.
> 1. Environment variables still need to be set in the heroku app that
> corresponds to this repository:
> 
> Currently:
> EMAIL_RECIPIENT = my email
> HEROKU_API_KEY = my api key
> MAIN_APP_NAME = vts-dev
> 
> Needs to be:
> EMAIL_RECIPIENT = a list-serv you (or me?) set up
> HEROKU_API_KEY = an api key that has access to the main app
> MAIN_APP_NAME = the name of the main production app in heroku
> 1. The command heroku run rake email_delivered_stories --app
>    vts-tracker-daily-digest needs to be added somewhere to the deployment
>    script.
> 
> I can do the (2), but (1) requires information I do not have access to.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/viewthespace/pivotal-deployment-emailer/pull/2#issuecomment-116860079
> .

## 

Sent from my mobile

**Pull request merged**